dashboard/apps/prototype/migrations/0013_auto_20160818_1040.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```

Comment:
this migration file doesn't look like it's generated by your changes. probably better removing it if that's the case.